supporting_content/metrics/amortizedcost.md

```diff
@@ -0,0 +1,71 @@
+## Supporting content
+
+### Column: Amortized Cost
+
+#### Example Scenarios
+
+
+
+### 7 Day Full upfront Reservation:
+
+##### On the day of purchase a line item exist for the upfront cost of the reservation as follows:
+
+
+| Charge period           | Billing period           | Charge Type | Units | Billed Cost | Amortized Cost |
+| ----------------------- | ------------------------ | ----------- | ----- | ----------- | -------------- |
+| Jan 1 2023 - Jan 1 2024 | Jan 1 2023 - Jan 31 2023 | Purchase    | 3     | 26.69589041 | 0              |
+
+
+###### Note 
+*Billed cost is showing the total cash outlay for the reservation. Amortization is negative in order to <insert reason here…I missed this part fully…I believe in order to negate any amortized cost>. Also note there is no initial RI fee representing the cost of the RI for the applicable period.*
```

Comment:
"Amortization is negative in order to <insert reason here…I missed this part fully…I believe in order to negate any amortized cost>."

- Is this necessary anymore, now that we've removed the amortization column?
- The reasoning is that the the record represents a capitalization, which will make the meaning of the summed column the net amortization/capitalization over the period
- Also, it's negative by definition because Billed Cost + Amortization = Amortized Cost